### ISSUE TITLE
Attendre quelque part les promesses issues de notify

### ISSUE THREAD

**Issue by alice-telescoop** [betagouv/api-subventions-asso#2180]
Sinon le processus peut s'arrêter sans qu'elles ne soient résolues (par exemple dans les migrations)

**Comment by VSerain** [comment]
Similaire à https://github.com/betagouv/api-subventions-asso/issues/2721 ?

**Comment by alice-telescoop** [comment]
Oui, mais pas tout à fait le même cadre et il faut bien gérer les deux situations (cli/cron de l'autre ticket et migrations de celui-ci)

**Comment by alice-telescoop** [comment]
(il faut merger les deux mais je sais pas lequel close et lequel compléter ><)
